.github/l10n/check_l10n_issues.py

```diff
@@ -8,11 +8,121 @@
 import xml.etree.ElementTree as etree
 import json
 import os
+import subprocess
 import sys
+import tempfile
 
 script_folder = os.path.abspath(os.path.dirname(__file__))
 vpn_root_folder = os.path.realpath(os.path.join(script_folder, os.pardir, os.pardir))
 
+def fileContentsAsJSON(filepath):
+    try:
+        with open(filepath, 'r') as file:
+            content = json.load(file)
+
+            if not content:
+                print(f"No content found in: {filepath}")
+                sys.exit(1)
+            return content
+    except FileNotFoundError:
+        print(f"File not found: {filepath}")
+        sys.exit(1)
+    except Exception as e:
+        print(f"An error occurred while loading {filepath}: {e}")
+        sys.exit(1)
+
+# Check if it uses shared strings. If so, pull the value for “title”, “subtitle”, then look within "blocks"
+def getSharedStringsInManifest(mainifest_contents):
+    shared_string_ids = []
+    # If it includes the key “message” and if within "message", there is a “usesSharedStrings” key that is true...
+    if "message" in manifest_contents and "usesSharedStrings" in manifest_contents["message"] and manifest_contents["message"]["usesSharedStrings"]:
```

Comment:
```suggestion
    if manifest_contents.get("message", {}).get("usesSharedStrings", {}).get("usesSharedStrings", None):
```

Using .get() with a default might be a bit more readable?

Reply:
Thanks! These last few months are the first time I've touched Python, so I'm still learning a lot. I'm shortening this even further; I think we just need `manifest_contents.get("message", {}).get("usesSharedStrings", False)`

Reply:
Yes, I duplicated the getter for `usesSharedStrings` at the end.